tools/worker-runner/cmd/start-worker/main.go

```diff
@@ -35,6 +37,12 @@ func main() {
 		os.Exit(1)
 	}
 
+	if opts["--short-version"].(bool) {
+		log.SetOutput(os.Stdout)
+		log.Printf("%s", internal.Version)
```

Comment:
Why not just `fmt.Printf("%s\n", internal.Version)` here, or even `fmt.Println(internal.Version)` ?

Reply:
no particular reason 😬 .. I forgot about fmt heh.. thanks :)